laske_export/tests/test_exporter.py

```diff
@@ -185,7 +185,7 @@ def test_send_invoices_to_laske_command_handle(
     assert len(mail.outbox) == 1
 
     export_mail = mail.outbox[0]
-    assert "Failed" in export_mail.body
+    assert ("epäonnistui" in export_mail.body) or ("Failed" in export_mail.body)
```

Comment:
This should be add to own commit. How this can happen?

Reply:
This test fails for me locally as the email body is in Finnish. I've reverted this line to only check for the English string as Travis doesn't use translations. I tried to force the language using `activate("en-en")`, but didn't see an effect, so maybe we'll leave it like this?